src/main/java/racingcar/step3/Car.java

```diff
@@ -0,0 +1,18 @@
+package racingcar.step3;
+
+import java.util.Random;
+
+class Car {
+    private int position = 0;
+
+    public void move() {
+        Random random = new Random();
+        if (random.nextInt(10) >= 4) {
+            position++;
```

Comment:
증감 연산자보다는 명확하게 표시하는 것이 더 좋습니다.

```suggestion
            position += 1;
```
증감연산자를 1개만 사용할 때는 이해하는데에는 문제가 없지만,
로직이 추가되고 증감연산하는 변수들이 늘어난다면 점점 복잡해질 수 있습니다.